README.md

```diff
@@ -338,6 +338,8 @@ API | Description | Auth | HTTPS | CORS |
 ### Dictionaries
 API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
+| [Chinese Character Web API](http://ccdb.hemiola.com/) | Chinese character definitions and pronunciations | No | No | No |
+| [Jisho](https://jisho.org/forum/54fefc1f6e73340b1f160000-is-there-any-kind-of-search-api) | Beta API for Jisho.org with Japanese language definitions | No | No | No |
```

Comment:
I didn't find the documentation for this API, the link you provided is a discussion.

If there is no documentation, this entry must be removed. Only properly documented APIs are accepted for listing here. 😅

Reply:
Here I refer to the Jisho API

Reply:
Got it. I swapped out its entry with the Ctext API, which is properly documented.